ci/aws/aws-cluster.lokocfg.envsubst

```diff
@@ -66,8 +69,13 @@ EOF
     disk_size     = 30
     instance_type = "t2.small"
     spot_price    = "0.01"
-    labels        = "testing.io=yes,roleofnode=testing"
-    taints        = "nodeType=storage:NoSchedule"
+    labels        = {
+      "testing.io" = "yes",
+      "roleofnode" = "testing",
+    }
+    taints        = {
+      "nodeType" = "storage:NoSchedule"
+    }
```

Comment:
Can you add a test to verify that this taint is taking effect on the node object? We have a mechanism for testing labels but we don't have any for the taints.

Reply:
yes we have `test/components/kubernetes/kubelet_taint_test.go`

Reply:
Sorry my bad for being ignorant about it.

Reply:
No worries.